### PR TITLE
update(zingchart): include default export

### PR DESCRIPTION
**Pull request by jeanettephung**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by jeanettephung — COMMENTED** [review]
Approve

**Comment by zingsoftadmin** [comment]
Approve

**Review by zingsoftadmin — APPROVED** [review]

**Review by zingsoftadmin — APPROVED** [review]
Approve

**Review by jeanettephung — COMMENTED** [review]
Approve

File: types/zingchart/index.d.ts [hunk 1]
```diff
@@ -4,4 +4,6 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.3
 // Minimum TypeScript Version: 3.7
-export * from './es6';
+import { zingchart } from './es6';
+
+export default zingchart;
```
Comment (andrewbranch) [inline types/zingchart/index.d.ts:9]: This is not correct. This file must correspond to https://unpkg.com/browse/zingchart@2.8.9/index.js, which is a CJS module, not an ES module. It should be `export = zingchart`, and users can use a default import by enabling `esModuleInterop` in their own projects. In the future, TypeScript will support separate definitions for CJS and ESM importers, but for now, you need to model the `main` entrypoint in this file.

**Review by andrewbranch — CHANGES_REQUESTED** [review]

**Review by jeanettephung — COMMENTED** [review]
Approved

File: types/zingchart/tslint.json [hunk 2]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "dtslint/dt.json"
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "npm-naming": [true,{"mode":"code","errors":[["NoDefaultExport",false]]}]
+  }
```
Comment (andrewbranch) [inline types/zingchart/tslint.json:5]: You should be able to delete this now.

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/zingchart/tslint.json (hunk 2 above)
Comment (jeanettephung) [reply types/zingchart/tslint.json:5]: Ah, thanks for getting back to me so quickly. I will apply this change.

**Review by andrewbranch — APPROVED** [review]
Thanks!

**Comment by jeanettephung** [comment]
Ready to merge

**Pull request merged**